cvat-core/src/api-implementation.ts

```diff
@@ -328,15 +328,23 @@ export default function implementAPI(cvat: CVATCore): CVATCore {
         return cloudStorages;
     });
 
-    implementationMixin(cvat.organizations.get, async (filter) => {
+    implementationMixin(cvat.organizations.get, async (filter, fullResponseData = false) => {
         checkFilter(filter, {
             search: isString,
             filter: isString,
+            page: isInteger,
+            page_size: isInteger,
         });
 
-        const organizationsData = await serverProxy.organizations.get(filter);
-        const organizations = organizationsData.map((organizationData) => new Organization(organizationData));
-        return organizations;
+        const organizationsData = await serverProxy.organizations.get(filter, fullResponseData);
+        if (fullResponseData) {
+            return {
+                ...organizationsData,
+                results: organizationsData.results.map((organizationData) => new Organization(organizationData)),
+            };
+        }
+
+        return organizationsData.map((organizationData) => new Organization(organizationData));
```

Comment:
Adding a parameter like `fullResponseData`  makes the `organizations.get` call inconsistent in terms of return values. Instead, I’d suggest always returning the organization data along with some additional metadata.

We already have the `PaginatedResource` type, which allows us to return something like `Object.assign(invitations, { count: invitationsData.count })`. We could create a new type extending from that, to include fields like `count` and `next`, if such metadata is needed on the CVAT UI side.